pkg/controller/installation/products/codeready/reconciler.go

```diff
@@ -149,12 +152,12 @@ func (r *Reconciler) reconcileBackups(ctx context.Context, inst *v1alpha1.Instal
 	return v1alpha1.PhaseCompleted, nil
 }
 
-func (r *Reconciler) reconcilePostgresSecret(ctx context.Context, serverClient pkgclient.Client) error {
+func (r *Reconciler) reconcilePostgresSecret(ctx context.Context, inst *v1alpha1.Installation, serverClient pkgclient.Client) error {
```

Comment:
It is failing to find the backup secret, causing the installation to remain in progress for me.

Reply:
I have a PR that automates the creation of s3-credentials secret but it should make any difference in the installation of the products

Reply:
Successful for me 
```
time="2019-11-22T15:28:43Z" level=info msg="reconciling subscription integreatly-codeready-workspaces from channel integreatly in namespace: integreatly-codeready-workspaces"
time="2019-11-22T15:28:45Z" level=info msg="creating subscription in ns if it doesn't already exist: integreatly-codeready-workspaces"
time="2019-11-22T15:28:45Z" level=info msg="creating required custom resources in namespace: integreatly-codeready-workspaces"
time="2019-11-22T15:28:45Z" level=info msg="checking keycloak client exists for che"
time="2019-11-22T15:28:45Z" level=info msg="codeready postgres component backup secret successfully reconciled"
time="2019-11-22T15:28:45Z" level=info msg="reconciling backups: codeready"
time="2019-11-22T15:28:45Z" level=info msg="codeready-workspaces has reconciled successfully"
```

Reply:
Seems to be reconciling for me as well